code_generator/Templates/UnityComponentConversionGenerator.tt

```diff
@@ -108,7 +108,9 @@ namespace <#= qualifiedNamespace #>
                 Unity.Entities.Entity entity;
                 if (!view.TryGetEntity(op.EntityId.Id, out entity))
                 {
-                    Debug.LogErrorFormat(TranslationErrors.OpReceivedButNoEntity, op.GetType().Name, op.EntityId.Id);
+                    view.LogDispatcher.HandleLog(new LogEvent("Entity not found during OnAddComponent in <#= componentDetails.ComponentName #>Translation.", LogType.Error)
```

Comment:
Could the LogDispatcher be cached on the translator during construction?

Reply:
I don't think so given that its possible for users to create their own LogDispatcher for each worker for example.

Reply:
Each worker gets their own Translation instance, so that should be fine

Reply:
Ahh yes I can get how to do it now, thanks. Changed :)